### PR TITLE
Create

### PR DESCRIPTION
**Pull request by katurian**
added form template and create_movie() logic

### CODE REVIEW

File: templates/create_movies_form.html [hunk 1]
```diff
@@ -5,5 +5,35 @@
 {% block body %}
 <h1 class="mb-5">Create Movie Rating</h1>
 <p class="mb-3">Create a movie rating below</p>
+<head>
+  <title>Event Registration</title>
+  <script src="https://cdn.tailwindcss.com"></script>
+</head>
```
Comment (JuanPablo2655) [inline templates/create_movies_form.html]: Probably best to move it in `_layout.html`

**Review by JuanPablo2655 — CHANGES_REQUESTED** [review]

**Comment by katurian** [comment]
Okay, will do.

File: tests/unit/test_create_movie.py [hunk 2]
```diff
@@ -1 +1,14 @@
 # TODO: Feature 2
+from src.models.movie import Movie
+from src.repositories.movie_repository import get_movie_repository
+
+movies = get_movie_repository()
+movies._db.__init__()
```
Comment (JuanPablo2655) [inline tests/unit/test_create_movie.py]:
```suggestion

```

File: tests/unit/test_create_movie.py [hunk 3]
```diff
@@ -1 +1,14 @@
...
+
+def test_create_movie():
+    currentMovies = movies.get_all_movies()
+    movies._db.__init__()
```
Comment (JuanPablo2655) [inline tests/unit/test_create_movie.py]:
```suggestion
    movies.__init__()
    currentMovies = movies.get_all_movies()
```

File: tests/unit/test_create_movie.py [hunk 4]
```diff
@@ -1 +1,14 @@
...
+    get_movie_repository.create_movie('Mulholland Drive', 'David Lynch', 5)
+    assert len(currentMovies) == 1
+    get_movie_repository.create_movie('Wings', 'Larisa Shepitko', 5)
```
Comment (JuanPablo2655) [inline tests/unit/test_create_movie.py]:
```suggestion
    movies.create_movie('Mulholland Drive', 'David Lynch', 5)
    assert len(currentMovies) == 1
    movies.create_movie('Wings', 'Larisa Shepitko', 5)
```

**Review by JuanPablo2655 — CHANGES_REQUESTED** [review]
I suggest testing before committing any changes

**Review by JuanPablo2655 — APPROVED** [review]
LGTM

**Pull request merged**